grammar.md

```diff
@@ -23,7 +23,7 @@ char               = (* Any UTF-8 code-point *)
 char_ascii         = 'a'...'z'
                    | 'A'...'Z'
 
-identifier         = char_ascii [ { char_ascii | digit | '_' | '!' | '?' } ]
+identifier         = char_ascii [ char_ascii | digit | '_' | '!' | '?' ]
```

Comment:
The file specifies both as being "zero or more", so the table has to be changed.